src/common/services/casApi.service.js

```diff
@@ -0,0 +1,69 @@
+import angular from 'angular';
+import 'angular-environment';
+import {Observable} from 'rxjs/Observable';
+import 'rxjs/add/observable/from';
+import 'rxjs/add/operator/map';
+import 'rxjs/add/operator/do';
+
+let serviceName = 'casApiService';
+
+class CasApi {
+
+  /*@ngInject*/
+  constructor( $http, envService ) {
+    this.$http = $http;
+    this.envService = envService;
+  }
+
+  http( config ) {
+    config.params = config.params || {};
+    if ( config.followLocation ) {
+      config.params.followLocation = true;
```

Comment:
I'll yank it out when I remove casApi.service in next PR. You can at least login now.